changelog.md

```diff
@@ -1,41 +1,39 @@
+* Added function to compute viscosity of water at specific temperature. Please refer to: [force calibration](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/force_calibration.html) for more information.
 * Added `CorrelatedStack.get_image()` to get the image stack data as an `np.ndarray`.
-* Allow setting custom slider ranges for the algorithm parameters in the kymotracker widget. See: [kymotracker widget](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/kymotracking.html#using-the-kymotracker-widget).
+* Allow setting custom slider ranges for the algorithm parameters in the kymotracker widget. Please refer to [kymotracker widget](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/kymotracking.html#using-the-kymotracker-widget) for more information.
 * Added parameters describing the inferred driving peak (`driving_amplitude`, `driving_frequency`, `driving_power`) when performing active force calibration to `CalibrationResults`.
-* Added function (`Scan.frame_timestamp_ranges()`) to obtain start and stop timestamp of each frame in a `Scan`. See: [Confocal images](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/images.html). 
+* Added function `Scan.frame_timestamp_ranges()` to obtain the start and stop timestamp of each frame in a `Scan`. Please refer to [Confocal images](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/images.html) for more information. 
 
 #### Improvements
 
 * Fixed issue in force calibration where the analytical fit would sometimes fail when the corner frequency is below the lower fitting bound. What would happen is that the analytical fit resulted in a negative term of which the square root was taken to obtain the corner frequency. Now this case is gracefully handled by setting the initial guess halfway between the lowest frequency in the power spectrum and zero.
-* Fixed bug that would implicitly convert `Kymograph` and `Scan` `timestamps` to floating point values. Converting them to floating point values leads to a loss of precision. For more information see [files and channels](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/file.html#channels) for more information.
 * Fixed issue that led to `DeprecationWarning` in the kymotracker widget.
 * Fixed error in kymotracking documentation. The tutorial previously indicated the incorrect number of samples for `sample_from_image`.
 
 #### Bug fixes
 
+* Fixed bug that would implicitly convert `Kymograph` and `Scan` `timestamps` to floating point values. Converting them to floating point values leads to a loss of precision. For more information see [files and channels](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/file.html#channels) for more information.
 * Fixed bug where color-aligned data was returned from `TiffFrame.data` although alignment was not requested (e.g., `CorrelatedStack("filename.tiff", align=False)`). This bug was introduced in `v0.10.1`.
```

Comment:
I figured bugs are places where the code did not produce an expected result. In the analytical fit failing case, it just produced an error. It didn't produce a bad result.

But yeah, where to draw that line seems a bit arbitrary in retrospect. I'll just group these together as bug fixes.

Reply:
It could just be about the phrasing. To me, saying "fixed" means it was something unintended (bug). You can definitely keep the improvements section, but then the entries need to be phrased accordingly: say "improved", "X now does Y", and similar.